### PR TITLE
Open a pull request to get code review

### PR DESCRIPTION
**Pull request by pulgosodog**
New branch from the main to add features: 

Create a new GitHub repo.
Create a new branch according to the GitHub flow rules.
Add a .gitignore file.
Add node_modules to gitignore file.
Set up linter for HTML and CSS.
Add a README.md.
Add a index.html, style.css and a test.md (this last one marked to be ignored on .gitignore). files
.

### CODE REVIEW

File: test.md
Comment (mukhammadrizooff) [inline test.md:1]: - [ ] [REQUIRED] Good job so far! This file should be not displayed on your code base, kindly remove it 🙏🏼 

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @pulgosodog, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Correct GitHub Flow ✔️ 
- Good implementations ✔️ 
- Descriptive commit history ✔️ 

## Required Changes ♻️
- [ ] [REQUIRED] Good job so far! Please add descriptive description to your `Pull Request` summary 🙏🏼 

image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: test.md
Comment (pulgosodog) [reply test.md:1]: My apologies, I though it was going to get removed after I update the .gitignore even though I already commited the test.md, but I just realize that that takes place locally.

Comment (pulgosodog) [reply test.md:1]: Addressed

**Review by AdnanOlarmmi — APPROVED** [review]
Hi @pulgosodog,

This is Adnan Is-haq, your beloved code reviewer, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Approved office man

## Highlights
- fixed required changes.
## Optional suggestions
_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**